mlflow/server/js/src/experiment-tracking/components/CompareRunView.tsx

```diff
@@ -383,12 +387,15 @@ class CompareRunView extends Component<CompareRunViewProps, CompareRunViewState>
       },
     ];
     return rows.map(({ key, title, data }) => (
-      <tr key={key}>
-        <th scope="row" className="head-value sticky-header" css={colWidthStyle}>
+      <TableRow key={key} className="compare-table-row">
+        <TableCell
```

Comment:
i think there is also a `TableHeader` which is supposed to be equivalent to `th`
```suggestion
        <TableHeader
```